File: erezervisi_api/eRezervisi/eRezervisi.Common.Dtos/Storage/FileDetails.cs
namespace eRezervisi.Common.Dtos.Storage
{
    public class FileDetails
    {
        public byte[] Bytes { get; set; } = null!;
        public string FileName { get; set; } = null!;
        public string ContentType { get; set; } = null!;
    }
}

File: erezervisi_mobile/erezervisi_mobile/lib/models/responses/storage/file_details.dart
import 'dart:typed_data';

class FileDetails {
  late Uint8List bytes;
  late String fileName;
  late String contentType;

  FileDetails(
      {required this.bytes, required this.fileName, required this.contentType});

  factory FileDetails.fromJson(Map<String, dynamic> json) {
    return FileDetails(
        bytes: json['bytes'],
        fileName: json['fileName'],
        contentType: json['contentType']);
  }
}
